src/sif/recost.cc

```diff
@@ -121,8 +121,11 @@ void recost_forward(baldr::GraphReader& reader,
     Cost transition_cost = node ? costing.TransitionCost(edge, node, label) : Cost{};
     // update the cost to the end of this edge
     uint8_t flow_sources;
+    const int64_t seconds_from_now =
+        int64_t(offset_time.seconds_from_now) * (offset_time.negative_seconds_from_now ? -1 : 1);
```

Comment:
yep i wondered what you would want to do here. i suppose one could do fading on either side of "now" and just ignore the sign, but i see you took the other approach. i think it would be interesting to actually allow it though. i dont have a strong opinion on it so what you did is fine with me.

Reply:
your idea sounds useful! Also it requires less operations so I modified this logic.